doc/getting_started/getting_started_go.rst

```diff
@@ -245,3 +245,197 @@ To send bare Lua code for execution, use ``Eval``:
+
+    *   -   License
+        -   BSD 2-Clause
+        -   MIT
+        -   BSD 2-Clause
+
+    *   -   Last update
+        -   2021
+        -   2021
+        -   2021
+
+    *   -   Known Issues
+        -   `#50 <https://github.com/tarantool/go-tarantool/issues/50>`_,
+            `#89 <https://github.com/tarantool/go-tarantool/issues/89>`_,
+            `#94 <https://github.com/tarantool/go-tarantool/issues/94>`_
+        -   `#41 <https://github.com/viciious/go-tarantool/issues/41>`_
+        -   No known issues
+
+    *   -   Documentation
+        -   README with examples, API described in code comments
+        -   README with examples, code comments
+        -   README with examples
+
+    *   -   Testing / CI / CD
+        -   GitHub Actions
+        -   Travis CI
+        -   GitHub Actions
+
+    *   -   Github Stars
+        -   127
+        -   43
+        -   12
+
+    *   -   Static Analysis
+        -   No
+        -   golint
+        -   golangci-lint
+
+    *   -   Packaging
+        -   go get
+        -   go get
+        -   go get
+
+    *   -   Code coverage
+        -   No
+        -   No
+        -   No
+
+    *   -   msgpack driver
+        -   `vmihailenco/msgpack/v2 <https://github.com/vmihailenco/msgpack/tree/v2>`_ (`#124 <https://github.com/tarantool/go-tarantool/issues/124>`_)
+        -   `tinylib/msgp <https://github.com/tinylib/msgp>`_
+        -   `vmihailenco/msgpack/v5 <https://github.com/vmihailenco/msgpack/tree/v5>`_
+
+    *   -   Async work
+        -   Yes
+        -   Yes
+        -   Yes
+
+    *   -   Batching
+        -   No
+        -   No
+        -   No
+
+    *   -   Schema reload
+        -   Yes (manual pull)
+        -   Yes (manual pull)
+        -   Yes (manual pull)
+
+    *   -   Space / index names
+        -   Yes
+        -   Yes
+        -   Yes
+
+    *   -   Tuples as structures
+        -   Yes (structure and marshall functions must be predefined in Go code)
+        -   No
+        -   Yes (structure and marshall functions must be predefined in Go code)
+
+    *   -   Access tuple fields by names
+        -   Only if marshalled to structure
+        -   No
+        -   Only if marshalled to structure
+
+    *   -   `SQL <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`_ support
+        -   No (`#62 <https://github.com/tarantool/go-tarantool/issues/62>`_)
+        -   No (`#18 <https://github.com/viciious/go-tarantool/issues/18>`_, closed)
+        -   No
+
+    *   -   `Interactive transactions <https://www.tarantool.io/en/doc/latest/book/box/stream/>`_
+        -   No (`#101 <https://github.com/tarantool/go-tarantool/issues/101>`_)
+        -   No
+        -   No
+
+    *   -   `Varbinary <https://www.tarantool.io/en/doc/latest/book/box/data_model/>`_ support
+        -   Yes (with in-built language tools)
+        -   Yes (with in-built language tools)
+        -   No (decodes to string by default, see `#6 <https://github.com/FZambia/tarantool/issues/6>`_)
```

Comment:
As mentioned in issue varbinary is supported - so I think Yes here with a link to issue.

Reply:
See https://github.com/tarantool/doc/pull/2484#discussion_r775405382 above.

> Decoding to a Unicode string means that varbinary in not supported properly. If your data in not Unicode you'll occasionally see errors or found the data corrupted (with replacement char) on the client side. So the answer is 'No' in practice.
>
> We can mention that we can do so in a custom decoder into a structure.

Reply:
See https://go.dev/blog/strings

> It’s important to state right up front that a string holds arbitrary bytes. It is not required to hold Unicode text, UTF-8 text, or any other predefined format. As far as the content of a string is concerned, it is exactly equivalent to a slice of bytes.

So I believe there is no problem with varbinary content if you just use `[]byte(varbinary_string)`.

Reply:
This changes everything. Thanks for the clarification!

Reply:
Reworked to positive